resources/puppetlabs/puppetdb/pql/pql-grammar-experimental.ebnf

```diff
@@ -121,10 +119,19 @@ sqstring = (<singlequote>, stringwithoutsinglequotes, <singlequote>);
 <stringwithoutsinglequotes> = #'(?:[^\'\\]|\\.)*';
 <singlequote> = "'";
 
-(* Modifiers *)
-<modifier> = groupby;
+(* Group By *)
+<groupbyclause> = groupby;
 groupby  = <'group by '>, fieldlist;
 
+(* Paging *)
+<pagingclause> = limit | offset | orderby;
+limit = <'limit '>, integer;
+offset = <'offset '>, integer;
+orderby = <'order by '>, orderparam, [orderparamlist];
```

Comment:
these three will only match exactly the space character; it would be better to use the `<whitespace>` non-terminal below.